inc/tools.php

```diff
@@ -227,6 +227,30 @@ function <?php echo esc_html( $callback ); ?>() {
  */
 function cptui_get_single_taxonomy_registery( $taxonomy = [] ) {
 
+	$might_need_to_resave = false;
+
+	$must_have_keys = ['object_types', 'rewrite', 'rewrite_slug', 'name',
+	'rewrite_withfront', 'rewrite_hierarchical', 'public', 'publicly_queryable',
+	'show_in_quick_edit', 'show_tagcloud', 'show_in_menu', 'show_ui', 'rest_base',
+	'show_in_nav_menus', 'show_in_rest', 'rest_controller_class', 'label',
+	'singular_label', 'meta_box_cb', 'default_term', 'singular_label',
+	'show_in_graphql', 'sort', 'query_var', 'show_ui', 'show_admin_column',
+	'hierarchical', 'graphql_single_name', 'graphql_plural_name'];
+
+	foreach( $must_have_keys as $key ) {
+		if ( ! array_key_exists($key, $taxonomy) ){
+			$taxonomy[$key] = '';
+
+			if ( $might_need_to_resave == false){
+				$might_need_to_resave = true;
+			}
+		}
+	}
+
+	if ( ! array_key_exists('labels', $taxonomy) ){
```

Comment:
Another nitpick, but spacing around our functions and logic statements. Her and above into the foreach loop above around line 241.

I know Alok is also working on his own PHPCS review so best to not introduce new spots when able.

Reply:
I'll run phpcs to check my added code